### PR TITLE
Create modernization-definition.md

### PR DESCRIPTION
**Pull request by ijmitch**
I added the initial paragraphs to `modernization-definition.md` but was unsure whether the other pieces should fit in the same place.

From the Word doc...

---
**Setup**:

-   Need to start with what is the PURPOSE of Mainframe Modernization

    -   Organizations modernize to continuously improve their business
        and build competitive advantage.

        -   Why start with this?

            -   Move people away from thinking that Modernization is
                _just_ tech -- it's about better business outcomes.

            -   Tech is ever changing -- will always have new tech --
                what is new now will be old

    -   ALL platforms and applications have value -- and that when it
        comes to technology -- you must extract the most value from
        _ALL_ your existing investments\...

        -   It's not about everything has to be on one platform -- it's
            a Hybrid world..

    -   Understand the risk and returns of modernization - what are the
        tradeoffs

**Scope:**

-   Modern IT methods span across these domains to support a hybrid
    world:

    -   AppDev

    -   Data Layer

    -   IT Operations and Infrastructure

    -   Security

    -   Integration & Interoperability

-   How to approach modernization (people, process, tech)

    -   Start with a best fit approach that meets business goals

    -   Creating a culture that embraces change, learning new skills,
        and collaboration

    -   IT Modernization requires:

        -   Open and integrated tech

        -   Tools and UI that reflect common/consistent ways of doing
            work and support modernization

        -   Automation everywhere

        -   People and Process changes

### CODE REVIEW

**Comment by rpomp** [comment]
To be clear, above is original OUTLINE version, not what was in word doc #4 

See below:
Objective:   Define the term mainframe modernization for use in the OMP Working Group

Defining Paragraph: 

Businesses are digitally transforming to better serve their customers, reduce time to market, and increase their ability to respond quickly to market changes, consumer behavior, and new regulations. As businesses transform, they are imposing significant demands on existing mainframe applications and data, driving the need for modernization to better achieve business outcomes.

It’s often not a question of if you need to modernize mainframe applications, only a question of how. The OMP Working Group embraces a hybrid cloud approach to mainframe modernization with the goal of helping business improve business agility, optimize costs, achieve non-functional requirements, while leveraging existing investments. We will provide a framework for modernization that spans applications, data, operations, interoperability, and security. We encourage looking at the business needs first, then take an ROI driven approach to help assess costs, business risks, and tradeoffs. There is rarely a “one-size fits all” approach to anything in the IT industry. By providing a framework we intend to feature the range of options available to help you to achieve your mainframe modernization objectives.

**Comment by ijmitch** [comment]
@rpomp - yes, and the two paras in the comment above are what are in the commit of a new file called `modernizationp-definition.md` in the branch associated with this PR.

I only included the bulleted list in the initial comment of this PR both for the historic record and to see if anyone was going to suggest a further change to the two paragraphs.

So I think the PR (or more precisely the commit to the branch it's proposing to merge) is aligned to your intent.

**Comment by ijmitch** [comment]
you can see the file which is proposed https://github.com/openmainframeproject/wg-modernization/blob/add-definition-proposal/modernization-definition.md or click on the *Files changed* tab at the top of this thread.

**Comment by rpomp** [comment]
I did miss the call, however I was given a quick debrief.   I now understand @ijmitch why you posted the First Principles.  This is what the team agreed to.   Before we modify any definition, we need to go back to the Principles.  Also, at what point does this get locked down?

**Comment by ijmitch** [comment]
> at what point does this get locked down?

Well, we can discuss here and iterate on improvements via commits to the branch until we decide to merge the branch into `main` which would signal that we're happy with it.

**Comment by rpomp** [comment]
ok

**Comment by b-around** [comment]
Sharing some ideas that I think should be in our definition of modernization which probably should be split into two parts – what we understand as modernization and how we will address modernization

OUR PERCEPTION OF MODERNIZATION CONTEXT
•	Modernization is about enabling continuous evolution of business operations (products, services, process) in line with a company mission and customer needs
•	Modernization is enabled by technology changes (apps, data, platforms, infrastructure, Dev & Ops tooling) and organizational changes (structure, culture, ways of working, alignment of priorities)
•	Mainframe platforms typically support business critical operations and due to its historical criticality and longevity, not always have been through holistic or consistent approaches to modernization 
•	The above sometimes leads to an incorrect view that the only option for modernizing areas that depend on mainframe is the replacement of the platform 

OUR WE WILL ADDRESS MODERNIZATION DEFINITION AND OUR BOUNDARIES
•	The OMP Working Group aims to provide a clear articulation of options, principles and patterns for mainframe platforms modernization in the form of a framework that guides the modernization of technology components and organizational practices around mainframes
•	The framework considers business needs, business value & ROI, execution risks & trade-offs, technology options & organizational impact
•	The framework is not tied to specific mainframe platforms, vendor technologies, tools, or target states and rather focus on the approach, execution process and required capabilities
•	Because organizational context matters and "one-size fits all" approaches don’t exist, thus the framework, parameters and considerations should always be adjusted to a company reality

**Comment by b-around** [comment]
Sharing some ideas that I think should be in our definition of modernization which probably should be split into two parts – what we understand as modernization and how we will address modernization

OUR PERCEPTION OF MODERNIZATION CONTEXT
•	Modernization is about enabling continuous evolution of business operations (products, services, process) in line with a company mission and customer needs
•	Modernization is enabled by technology changes (apps, data, platforms, infrastructure, Dev & Ops tooling) and organizational changes (structure, culture, ways of working, alignment of priorities)
•	Mainframe platforms typically support business critical operations and due to its historical criticality and longevity, not always have been through holistic or consistent approaches to modernization 
•	The above sometimes leads to an incorrect view that the only option for modernizing areas that depend on mainframe is the replacement of the platform 

OUR WE WILL ADDRESS MODERNIZATION DEFINITION AND OUR BOUNDARIES
•	The OMP Working Group aims to provide a clear articulation of options, principles and patterns for mainframe platforms modernization in the form of a framework that guides the modernization of technology components and organizational practices around mainframes
•	The framework considers business needs, business value & ROI, execution risks & trade-offs, technology options & organizational impact
•	The framework is not tied to specific mainframe platforms, vendor technologies, tools, or target states and rather focus on the approach, execution process and required capabilities
•	Because organizational context matters and "one-size fits all" approaches don’t exist, thus the framework, parameters and considerations should always be adjusted to a company reality

**Comment by rpomp** [comment]
@b-around this is consistent with our First Principles that @ijmitch posted initially. I am fine with this direction.

**Comment by ijmitch** [comment]
OK, from a process point of view I'm going to progress this PR from being DRAFT so we can merge the current proposal and move on to further PRs to improve/refine it.

File: modernization-definition.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Defining Mainframe Modernization
+
+Businesses are digitally transforming to better serve their customers,
+reduce time to market, and increase their ability to respond quickly to
+market changes, consumer behavior, and new regulations. As businesses
+transform, they are imposing significant demands on existing mainframe
+applications and data, driving the need for modernization to better
+achieve business outcomes.
+
+It's often not a question of if you need to modernize mainframe
```
Comment (ijmitch) [inline modernization-definition.md:10]: Now that we're in the review cycle for this PR, it's appropriate to comment on the specific lines of this proposed text.

**Comment by ijmitch** [comment]
@MistyDecker I recommend that you approve this PR so it can be merged and we move on with refining it in further more specific PRs.

File: modernization-definition.md (hunk 1 above)
Comment (MistyDecker) [reply modernization-definition.md:10]: Ok boss.

**Review by MistyDecker — APPROVED** [review]
Happy now Ian??

File: modernization-definition.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+applications, only a question of how. The OMP Working Group embraces a
+hybrid cloud approach to mainframe modernization with the goal of
```
Comment (MistyDecker) [inline modernization-definition.md:12]: full range of approaches to mainframe modernization, including modernizing in place, hybrid cloud or moving to another platform.  The methods chosen need to focus on the goals of

**Pull request merged**